### ISSUE TITLE
Be able to execute an install hook when installing packages

### ISSUE THREAD

**Issue by bhou-crto** [criteo/command-launcher#35]
For some commands, it requires an additional "install" step after downloading, unzipping to local repository before it is ready to be executed by command launcher.

A possible solution is to add an install command in the package manifest:

```
{
  "pkgName": "test-pkg",
  "version": "1.0.0",
  "cmds": {
     ...
  },
  "installCmd": "{{.Cache}}/install.sh",
  "installCmdArgs": []
}
```

The install command will be called once a new version of the package is downloaded and unzipped to local repository

**Comment by bhou** [comment]
This feature is implemented as a `system` command `__setup__`. See: https://github.com/criteo/command-launcher/pull/110

**Comment by bhou** [comment]
Closed by #110
